src/preset_cli/cli/superset/sync/dbt/databases.py

```diff
@@ -78,6 +80,16 @@ def sync_database(  # pylint: disable=too-many-locals, too-many-arguments
             **meta,
         )
 
-    database["sqlalchemy_uri"] = connection_params["sqlalchemy_uri"]
+        database["sqlalchemy_uri"] = connection_params["sqlalchemy_uri"]
+
+    elif databases:
+        _logger.info("Found an existing database connection, using it")
+        database = databases[0]
+        database["sqlalchemy_uri"] = client.get_database(database["id"])[
```

Comment:
In case `--import-db` is not passed, I added this to fetch the `SQLAlchemy` URI from the Superset API instead. The password wouldn't be included, but I don't think this would be a problem since we're not updating the DB connection.